examples/kubernetes/README.md

```diff
@@ -0,0 +1,49 @@
+
+# Kubernetes
+
+This example provisions a Kubernetes cluster with 1 master node, 1 worker node, and a dedicated etcd node. Each node uses a static IP address on the local network.
+
+## Requirements
+
+First, follow the [examples](../README.md) documentation to setup client machines, the Config service, and the network environment.
```

Comment:
The ordering of these instructions are somewhat unclear:

The README says to look at the example after the config service, but before the network environment.
Then this says that it should be done after both the config service and the network environment. But per IRL discussion, the config service mount depends on the example being used.

May be easier to just have really linear instructions, even if it means duplicating some documentation per-example